### PR TITLE
CV and BERT Examples for NVidia Triton intergation for Amazon Sagemaker

### PR DESCRIPTION
**Pull request by maaquib**
*Issue #, if available:*

*Description of changes:* Adding example for using Nvidia Triton server on Sagemaker. Details in "New Amazon SageMaker integration" section of [blog](https://developer.nvidia.com/blog/fast-and-scalable-ai-model-deployment-with-nvidia-triton-inference-server/)

*Testing done:* Verified by running notebook e-2-e

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [X] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [X] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [X] I have tested my notebook(s) and ensured it runs end-to-end
- [X] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by shreyapandit** [comment]
@maaquib can you please format the notebooks as advised in the code formatting check, use

`black-nb -l 100 your_notebook_file.ipynb` for all notebook files you have changed, thanks!

File: sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb [hunk 1]
```diff
@@ -0,0 +1,681 @@
+{
```
Comment (jkroll-aws) [inline sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb:1]: <p>Looks like nested numbering is not supported. (In both notebooks)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (jkroll-aws) [inline sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    triton_image_uri = "195202947636.dkr.ecr.us-west-2.amazonaws.com/tritonserver:21.08-py3"
</pre><p>use <code>sagemaker.image_uris.retrieve()</code>. Account ID, region, and version tag cannot be hard-coded. (In both notebooks)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (jkroll-aws) [inline sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb:1]: <p>Do a find and replace for all markdown occurrences of <code>Sagemaker</code> and change to <code>SageMaker</code>	across both notebooks in the PR.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Comment (jkroll-aws) [inline sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    text = "Amazon SageMaker helps data scientists and developers to prepare, build, train, and deploy high-quality machine learning (ML) models quickly by bringing together a broad set of capabilities purpose-built for ML."
</pre><p>This sentence is defined twice in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3022/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Comment (maaquib) [reply sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb:1]: Seems to be working when rendered by github https://github.com/maaquib/amazon-sagemaker-examples/blob/triton/sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb

Not sure why it is not working in ReviewNB

Comment (jkroll-aws) [reply sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb:1]: Thanks for the link, you can disregard this comment given that it seems to be a ReviewNB issue.

Comment (maaquib) [reply sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb:1]: Done

Comment (maaquib) [reply sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb:1]: Done

Comment (maaquib) [reply sagemaker-triton/nlp_bert/triton_nlp_bert.ipynb:1]: <ul><li>Removed hard-coding</li><li>Adding logic to generate the uri based on region and <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/neo-deployment-hosting-services-container-images.html" target="_blank">account ids</a></li><li>Added error handling for unsupported regions</li></ul>

**Pull request merged**